packages/tailwindcss/src/design-system.ts

```diff
@@ -127,6 +137,14 @@ export function buildDesignSystem(theme: Theme): DesignSystem {
     compileAstNodes(candidate: Candidate) {
       return compiledAstNodes.get(candidate)
     },
+
+    printCandidate(candidate: Candidate) {
+      return printCandidate(designSystem, candidate)
+    },
+    printVariant(variant: Variant) {
+      return printVariant(variant)
+    },
```

Comment:
What's the value of adding this to the design system? I remember we moved it into the upgrade tool to avoid shipping this with everyone installing `tailwindcss`. Since this is part of a default exported object now, it can't be DCE-ed 🤔 
Are we planning on using this in other places?

Reply:
This is a bit for selfish reasons, but here are the reasons for it:

- We already have `parseCandidate` and `parseVariant` exposed
- I've been copying the `printCandidate` / `printVariant` for the internal debugger / extractor tools
- @thecrypticace mentioned he has a use case for it in intellisense
- The total size of these print functions is still very tiny
   - Without, the `dist/` folder we publish is 606 KB
   - With, the `dist/` folder we publish is 614 KB

Reply:
FINE